tarpc/src/rpc/transport/channel.rs

```diff
@@ -89,7 +89,8 @@ mod tests {
     use log::trace;
     use std::io;
 
-    #[tokio::test]
+    #[cfg(feature = "tokio1")]
```

Comment:
Tokio is in the dev dependencies right? Maybe this just needs to be `#[cfg(test)]`